README.md

```diff
@@ -51,14 +51,29 @@ inventory:
       # Run every minute
       schedule: '* * * * *'
 
-# This needs to be created separately, and contain the following keys:
+# This needs to be created separately (read instructions below), and contain the following keys:
 # - `HASHICORP_VAULT_TOKEN` - the hashicorp vault token to use
```

Comment:
we have `GITLAB_TOKEN` as well in the example, do we want to add it here and in the `Secret` below to remove any confusion?